Softeq.XToolkit.WhiteLabel.Forms/Navigation/RootFrameNavigationPage.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using Xamarin.Forms;
+
+namespace Softeq.XToolkit.WhiteLabel.Forms.Navigation
+{
+    public class RootFrameNavigationPage : NavigationPage
```

Comment:
Looks than in this case application can't customize navigation root

Reply:
We still have opportunity to create separate navigation page + view model or navigation page only and override BuildRootFrameNavigationPageTypeName in ViewLocator, but feels like in most cases application won't need that

Reply:
Agree!